docs/source/nn.rst

```diff
@@ -1,1001 +1,25 @@
-    :inherited-members:
-    :members:
-
-:hidden:`CustomFromMask`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autoclass :: torch.nn.utils.prune.CustomFromMask
-    :inherited-members:
-    :members:
-
-:hidden:`identity`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.identity
-
-:hidden:`random_unstructured`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.random_unstructured
-
-:hidden:`l1_unstructured`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.l1_unstructured
-
-:hidden:`random_structured`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.random_structured
-
-:hidden:`ln_structured`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.ln_structured
-
-:hidden:`global_unstructured`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.global_unstructured
-
-:hidden:`custom_from_mask`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.custom_from_mask
-
-:hidden:`remove`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.remove
-
-:hidden:`is_pruned`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction :: torch.nn.utils.prune.is_pruned
-
-:hidden:`weight_norm`
-~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.weight_norm
-
-:hidden:`remove_weight_norm`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.remove_weight_norm
-
-:hidden:`spectral_norm`
-~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.spectral_norm
-
-:hidden:`remove_spectral_norm`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.remove_spectral_norm
-
-
-.. currentmodule:: torch.nn.utils.rnn
-
-:hidden:`PackedSequence`
-~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.rnn.PackedSequence
-
-
-:hidden:`pack_padded_sequence`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.rnn.pack_padded_sequence
-
-
-:hidden:`pad_packed_sequence`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.rnn.pad_packed_sequence
-
-
-:hidden:`pad_sequence`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.rnn.pad_sequence
-
-
-:hidden:`pack_sequence`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.rnn.pack_sequence
-
-.. currentmodule:: torch.nn
-
-:hidden:`Flatten`
-~~~~~~~~~~~~~~~~~
-
-.. autoclass:: Flatten
-    :members:
-
-
-Quantized Functions
---------------------
-
-Quantization refers to techniques for performing computations and storing tensors at lower bitwidths than
-floating point precision. PyTorch supports both per tensor and per channel asymmetric linear quantization. To learn more how to use quantized functions in PyTorch, please refer to the :ref:`quantization-doc` documentation.
+.. toctree::
+   :maxdepth: 1
+
+   nn/nn_containers
```

Comment:
I'm not sure these should be alphabetized: it's important that parameters comes first and a lot of these were grouped based on relevancy (for example, convolution, padding, and pooling all relate to image processing, so they were adjacent to each other)

Reply:
Okay that makes sense to me. I just pushed new changes that revers the alphabetization